airflow/models.py

```diff
@@ -3368,6 +3416,37 @@ def __repr__(self):
     def id_for_date(klass, date, prefix=ID_FORMAT_PREFIX):
         return prefix.format(date.isoformat()[:19])
 
+    @provide_session
+    def refresh_from_db(self, session=None):
```

Comment:
about docstrings, I had pylint setup on landscape.io to only require a docstring if the method is 10+ lines which I thought was reasonable